configure.ac

```diff
@@ -300,7 +301,7 @@ fi
 if [test "$CUDA_H_PATH" ]; then
 	AC_DEFINE([HAVE_CUDA], [1], [Enable CUDA feature])
 	AC_DEFINE_UNQUOTED([CUDA_PATH], "$CUDA_H_PATH" , [Enable CUDA feature])
-	LIBS=$LIBS" -lcuda"
+	LIBS=$LIBS" -lcuda -lcudart"
```

Comment:
for some reason, when running ./configure with this change it wont find `cuMemGetHandleForAddressRange`, but when separating it into:
`LIBS=$LIBS" -lcuda"
AC_CHECK_LIB([cuda], [cuMemGetHandleForAddressRange], [HAVE_CUDA_CUMEMGETHANDLEFORADDRESSRANGE=yes], [HAVE_CUDA_CUMEMGETHANDLEFORADDRESSRANGE=no])
LIBS=$LIBS" -lcudart"
`

it will find `cuMemGetHandleForAddressRange`